app/src/main/java/com/example/android/sunshine/app/Utility.java

```diff
@@ -110,6 +110,25 @@ public static String getFriendlyDayString(Context context, long dateInMillis) {
     }
 
     /**
+     * Helper method to convert the database representation of the date into something to display
+     * to users.  As classy and polished a user experience as "20140102" is, we can do better.
```

Comment:
We could probably do better with the class name though. Utility isn't very descriptive. Also it looks like a potential god class in the making! How about splitting the weather-related utility functions out from the date-related ones and having classes like `DateUtils`, `WeatherIconUtils`, `LocationUtils` etc. And of course maybe consider `MerlinsBeard` instead of `NetworkUtils`....